user-management-oidc.md

```diff
@@ -0,0 +1,51 @@
+User Management - SSO/OIDC
+==========================
+
+> **Note**: OIDC capabilities are only included with a Kubecost Enterprise Subscription.
+
+## Helm configuration
+
+The OIDC integration in Kubecost is fulfilled via the helm configuration parameters `.Values.oidc` [as shown here](https://github.com/kubecost/cost-analyzer-helm-chart/blob/develop/cost-analyzer/values.yaml).
+
+## Supported identity providers
+
+Please refer to the following references to find out more about how to configure the Helm parameters to suit each OIDC identiy provider integration.
+
```

Comment:
There is an information gap here as we have no public Keycloak documentation to reference. I had a brief look over their pages and I couldn't find a valid resource to share. On the other hand, we have this [internal Keycloak doc](https://docs.google.com/document/d/10XtJhWlYD0TcjN-Tohk38eXWmUwJHJsomOti7uqiAJ8/edit#).

We might want to revisit this section altogether and deciding whether we want to share these external resources in our public docs in the first place, or to leave them internal to our Support team. 

Do you have any thoughts here? @thomasvn 
cc @kaelanspatel

Reply:
This is more of a product decision, tbh. While I'm in favor of making config as easy as possible, where did we land officially on making exact setup public? 

Maybe @AjayTripathy can say?

Reply:
I think it's best to document thoroughly and make the setup public but note that this is enterprise-only. That sound good?

Reply:
I would agree. We've documented Enterprise only features like that before. Need me to make an edit?

Reply:
@biancaburtoiu . I've added the Keycloak setup information. Could you help to validate its correctness?

Reply:
this lgtm, @kaelanspatel wdyt about these keycloak setup instructions refactored for the public docs?

Reply:
@biancaburtoiu @thomasvn LGTM! Tiny thing, I think adding "dummy" values (e.g. `clientID: "<your client ID>"`  vs `clientID: ""`) makes the helm config section clearer at a glance. Seeing "the `Values.oidc` for Keycloak should be as follows" and having a whole bunch of blank values threw me off for a second.

Obviously a nit, great stuff in any case.

Reply:
@biancaburtoiu @kaelanspatel. Thanks for the review! I've added in the dummy values, and also capitalized the dummy values to make it more pronounced that they need to be replaced.